cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp

```diff
@@ -160,12 +163,12 @@
 
 (defun send-object-action-parameter (action-inst object-designator)
   (let ((object-instance-id (write-to-string (desig::desig-prop-value object-designator :NAME))))
-    (send-rdf-query (convert-to-prolog-str action-inst) "knowrob:object" (convert-to-prolog-str object-instance-id))
+    (send-rdf-query (convert-to-prolog-str action-inst) "knowrob:objectActedOn" (convert-to-prolog-str object-instance-id))
     object-instance-id))
 
 (defun get-object-name (object-name)
```

Comment:
@seba90 Hey Sebastian, you're not using this function anywhere. Where was it supposed to be used?

Reply:
@seba90 Ah, found the place, actually right here on L162. Ok, it's an easy fix, I'll do it, so never mind.